quinn/src/tests.rs

```diff
@@ -159,17 +159,29 @@ fn export_keying_material() {
     };
 
     runtime.block_on(async move {
-        let outgoing_conn = endpoint
-            .connect(endpoint.local_addr().unwrap(), "localhost")
-            .unwrap()
-            .await
-            .expect("connect");
-        let incoming_conn = endpoint
-            .accept()
-            .await
-            .expect("endpoint")
-            .await
-            .expect("connection");
+        let outgoing_conn_fut = tokio::spawn({
+            let endpoint = endpoint.clone();
+            async move {
+                endpoint
+                    .connect(endpoint.local_addr().unwrap(), "localhost")
+                    .unwrap()
+                    .await
+                    .expect("connect")
+            }
+        });
+        let incoming_conn_fut = tokio::spawn({
+            let endpoint = endpoint.clone();
+            async move {
+                endpoint
+                    .accept()
+                    .await
+                    .expect("endpoint")
+                    .await
+                    .expect("connection")
+            }
+        });
+        let outgoing_conn = outgoing_conn_fut.await.unwrap();
+        let incoming_conn = incoming_conn_fut.await.unwrap();
```

Comment:
It'd be simpler to skip spawning and just `join` the two.

Reply:
Done.